p2p/net/session_cache.go

```diff
@@ -0,0 +1,90 @@
+package net
+
+import (
+	"errors"
+	"github.com/spacemeshos/go-spacemesh/p2p/p2pcrypto"
+	"sync"
+	"time"
+)
+
+// simple cache for storing sessions
+const maxSessions = 1024
```

Comment:
This value has a major implications on the behavior of the node, it should be in the configuration